File: Day1_Second Largest/second_largest.java
import java.util.*;

public class second_largest {

    static int getSecondLargest(ArrayList<Integer>arr) {
        int second_large = -1;
        int large = arr.get(0);
        for(int i=0;i<arr.size();i++) {
            if(arr.get(i) > large){
                second_large = large;
                large = arr.get(i);
            } 
            else if(arr.get(i) < large && arr.get(i) > second_large)
                second_large = arr.get(i);
        }
        return second_large;
    }
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the size and space-separated integers: ");
        ArrayList<Integer> arr = new ArrayList<>();

        int n = scanner.nextInt();
        for(int i=0;i<n;i++)
            arr.add(scanner.nextInt());
        System.out.println(getSecondLargest(arr));
        scanner.close();
    }
}

File: Day1_Second Largest/second_largest.py
# https://www.geeksforgeeks.org/batch/gfg-160-problems/track/arrays-gfg-160/problem/second-largest3735
def getSecondLargest(arr):
    # Code Here
    second_large = -1
    large = arr[0]
    for i in arr:
        if i > large:
            second_large = large
            large = i
        elif i < large and i > second_large:
            second_large = i
    return second_large


user_input = input("Enter space-separated integers: ")
arr = list(map(int, user_input.split()))
print(getSecondLargest(arr=arr))
